src/models/tortoise/__init__.py

```
@@ -0,0 +1,4 @@
+"""This module initialises the tortoise models package by importing all necessary model classes."""
+
+from .agencies import Agencies
```

Ruff (0.12.2)

`.agencies.Agencies` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)